src/main/java/mate/Main.java

```diff
@@ -0,0 +1,7 @@
+package mate;
+
+public class Main {
+    public static void main(String[] args) {
```

Comment:
Consider adding functionality to the main method or removing it if it is not needed. 🤔